DirectorySearcherLib/DirectorySearcher.cs

```diff
@@ -48,7 +48,7 @@ public static class DirectorySearcher
 
             try
             {
-                string graphRequest = String.Format(CultureInfo.InvariantCulture, "{0}/{1}/users?api-version={2}&$filter=mailNickname eq '{3}'", graphResourceUri, authResult.TenantId, graphApiVersion, alias);
+                string graphRequest = String.Format(CultureInfo.InvariantCulture, "{0}/v1.0/users?$filter=mailNickname eq '{3}'", graphResourceUri, authResult.TenantId, graphApiVersion, alias);
```

Comment:
>0}/v1.0/users?$filter=mailNickname eq '{3}' [](start = 84, length = 43)

you are only using param 0 and 3. consider rewriting this and remove the no longer needed info. Also you can use the $"" format to make it more readable?